.github/workflows/forum.yml

```diff
@@ -0,0 +1,60 @@
+name: Move to Typst Forum
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'to-forum'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Call Discourse API to create forum post
+        env:
+          ISSUE_BODY: ${{ github.event.issue.body }}
+          ISSUE_TITLE: ${{ github.event.issue.title }}
+        run: |
+          RAW="This topic was moved from GitHub issue ${{ github.event.issue.number }}. Please continue the conversation here.
+
+          Here is the original issue:
+          [quote=\"${{ github.event.issue.user.login }}\"]
+          $ISSUE_BODY
+          [/quote]
+
+          Browse the previous discussion at ${{ github.event.issue.html_url }}"
+          TITLE_JSON=$(jq -n --arg title "[#${{ github.event.issue.number }}] $ISSUE_TITLE" '$title')
+          RAW_JSON=$(jq -n --arg raw "$RAW" '$raw')
+
+          RESPONSE=$(curl -X POST "https://forum.typst.app/posts.json" \
+            -H "Api-Key: ${{ secrets.DISCOURSE_TOKEN }}" \
+            -H "Content-Type: application/json" \
+            -d "{
+              \"title\": $TITLE_JSON,
+              \"category\": 4,
+              \"external_id\": \"gh-${{ github.repository_owner }}-typst-${{ github.event.issue.number }}\",
+              \"raw\": $RAW_JSON
+            }")
+
+          echo $RESPONSE
```

Comment:
I wonder if the response could potentially contain any sensitive information by mistake. Worth thinking about as this would be public. Could also just not `echo` this and, instead, log negative responses coming from GitHub in the Typst server's side for us to inspect internally if needed.

Reply:
I changed this to only echo the values from the `error` array if present.